api/prometheus.go

```diff
@@ -50,61 +53,128 @@ func (cs ConsensusState) PrometheusMetric() []prometheus.Metric {
 				}
 				return 0
 			}(),
-			Timestamp: time.Now(),
 		},
 		{
-			Name:      "hostd_consensus_state_index",
-			Labels:    map[string]any{"id": cs.ChainIndex.ID},
-			Value:     float64(cs.ChainIndex.Height),
-			Timestamp: time.Now(),
+			Name:  "hostd_consensus_state_chain_index_height",
+			Value: float64(cs.ChainIndex.Height),
+		},
+		{
+			Name: "hostd_consensus_state_chain_index_height_B", //TODO: grafana cannot simultaneously display block height and colorize based of this synced label. the bustedware/grafana repo is working on a forked grafana that will one day make this work
```

Comment:
Change this back to `hostd_consensus_state_index` and remove any `_B` and `_C` metrics. `ID` is necessary as part of the sync state.